### ISSUE TITLE
[Question] Is there a way to disable websocket?

### ISSUE THREAD

**Issue by kud** [appbaseio/reactivesearch#208]
We only use ES via http(s) here, not via websocket. So I got errors like:

```
Firefox can’t establish a connection to the server at ws://domain.com//api/search/bills,amendments.
better_websocket.js:10
```

Is there a way to disable the websocket method?

**Comment by metagrover** [comment]
websockets are in place to ensure streaming (which only works with appbase.io). Currently, there is no way to avoid it. It's a non-consequent error though. It shouldn't affect anything. We'll possibly fix this in the coming releases.

**Comment by kud** [comment]
Thank you. :)

**Comment by metagrover** [comment]
Fixed in 2.2.0 🎉
